pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -182,80 +188,102 @@ func validateConstraintNotRepeat(path *field.Path, constraints []v1.TopologySpre
 
 // ValidateRequestedToCapacityRatioArgs validates that RequestedToCapacityRatioArgs are correct.
 func ValidateRequestedToCapacityRatioArgs(args config.RequestedToCapacityRatioArgs) error {
-	if err := validateFunctionShape(args.Shape); err != nil {
-		return err
+	var allErrs field.ErrorList
+	allErrs = append(allErrs, validateFunctionShape(args.Shape, field.NewPath("shape"))...)
+	allErrs = append(allErrs, validateResourcesNoMax(args.Resources, field.NewPath("resources"))...)
+	return allErrs.ToAggregate()
+}
+
+func validateFunctionShape(shape []config.UtilizationShapePoint, p *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if len(shape) == 0 {
+		allErrs = append(allErrs, field.Required(p, "at least one point must be specified"))
+		return allErrs
 	}
-	if err := validateResourcesNoMax(args.Resources); err != nil {
-		return err
+
+	for i := 1; i < len(shape); i++ {
+		if shape[i-1].Utilization >= shape[i].Utilization {
+			msg := fmt.Sprintf("utilization values must be sorted. Should < %s==%d", p.Index(i).Child("utilization").String(), shape[i].Utilization)
```

Comment:
I have some concerns here. If there are many unsorted elements, then "utilization values must be sorted." will be repeated many times.

Reply:
If you want, you could just identify the first one and break from the loop like before. But if you want to return all the errors I don't see a much better way to collect them other than maybe building a list of all the invalid fields and combining them into 1 error.

Reply:
Error on the first one is fine.

No need to mention the values

```golang
field.Invalid(p.Index(i).Child("utilization"), shape[i].Utilization, "utilization values must be sorted in increasing order")
```